electron-builder.yml

```diff
@@ -5,6 +5,7 @@ files:
   - "!**/node_modules/**"
   - "dist-electron"
   - "package.json"
+  - "!dist-electron/screenshots/**/*"
```

Comment:
Will it work as `- "!dist-electron/screenshots/**"`?

Reply:
it is recommended by electron to do it like that